nf-test.config

```diff
@@ -3,13 +3,10 @@ config {
     testsDir "."
 
     // nf-test directory including temporary files for each test
-    workDir "/tmp"
-
-    // location of library folder that is added automatically to the classpath
-    libDir "lib/"
+    workDir ".nf-test"
```

Comment:
I'd rather we use a $TMP_DIR that we set up to `.nf-test`

Reply:
I think both Adam and I have had trouble making the TMPDIR work

Reply:
Seems to work though (minus the closure)